src/store/redis/redis_nodeset.c

```diff
@@ -986,13 +986,15 @@ static void nodeset_cluster_check_event_callback(redisAsyncContext *ac, void *re
   int                         epoch;
   const char                 *err = NULL;
   
-  node_command_received(node);
+  node_command_time_finish(node, NCHAN_REDIS_CMD_CLUSTER_CHECK);
   
   if(reply == NULL) {
     //Node should have already disconnected
     return;
   }
   
+  node_command_received(node);
```

Comment:
You found a legit potential bug caused by a weird merge.

However, it would not manifest because when `reply == NULL` it means the node has disconnected, and the `command_received` tracking gets reset.

So it's wrong but luckily gets unwronged by cleanup code further down the way.

Fixed in 9c181687.